src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -37,69 +39,72 @@ If you're more adventurous or want to get the latest changes of the xk6-browser
 
 <InstallationInstructions extensionUrl="github.com/grafana/xk6-browser"/>
 
+## Launch
+
+The first step is to launch a [Browser](/javascript-api/xk6-browser/browser). After the browser starts, you can interact with it using the [browser-level APIs](#browser-level-apis). The only accepted browser type is currently `chromium`.
```

Comment:
Since we changed the API in https://github.com/grafana/xk6-browser/pull/462, let's update the last sentence here, and the `launch()` documentation below.

Actually, since we now export browser names directly, it might make sense to have separate browser sections here, instead of the "Launch" `h2`. This would make it clear that only `chromium` is currently supported, and also allow us to expand the docs once support for other browsers is added.

Reply:
Sounds good to me. Can i do these changes in a separate PR? I'm going to work on the docs for the https://github.com/grafana/xk6-browser/pull/462 change in one single PR.